classes/dpkg-bazel.bbclass

```diff
@@ -12,20 +12,30 @@
 inherit dpkg
 
 DEPENDS += "bazel-bootstrap"
+BAZEL_FETCH_DEPENDS += "bazel-bootstrap bazel-bootstrap-data git symlinks"
+
+# ccache + bazel is broken
+USE_CCACHE="0"
 
 # fetch all bazel deps and add to package sources
 bazel_fetch() {
-    dpkg_do_mounts
     E="${@ isar_export_proxies(d)}"
-    sudo chroot --userspec=$( id -u ):$( id -g ) ${BUILDCHROOT_DIR} /bin/bash -c 'cd ${PP}/${PPS} && rm -rf local && ./debian/rules local'
-    dpkg_undo_mounts
+    echo "SCHROOT: ${SBUILD_CHROOT}"
+    schroot -d / -c ${SBUILD_CHROOT} -u root -- sh <<EOF
+        echo ${ISAR_APT_REPO} > /etc/apt/sources.list.d/isar_apt.list;
+        apt-get -y -q update
+        apt-get -y -q -o Debug::pkgProblemResolver=yes \
+                    --no-install-recommends --allow-downgrades \
+                    install ${BAZEL_FETCH_DEPENDS}
+        cd ${PP}/${PPS} && ./debian/rules local
+EOF
 
     # generate orig archive, compress in parallel
-    tar -c -I 'xz -1 -T0' -f ${WORKDIR}/${PN}_${PV}.orig.tar.xz -C ${S} --exclude=.git --exclude='./debian' .
-    # fix permissions of bazel cache
-    sudo chown -R $( id -u ):$( id -g ) ${WORKDIR}/.cache/bazel
+    tar -c -I 'xz -1 -T0' -f ${WORKDIR}/${PN}_${PV}.orig.tar.xz -C ${S} --exclude=.git --exclude=.env .
```

Comment:
We should add xz options similar to the ones from `imagetypes.bbclass` to avoid OOM in CI.